examples/website/3d-tiles/app.js

```diff
@@ -224,17 +224,17 @@ export default class App extends PureComponent {
       return null;
     }
 
-    const {ionAssetId, ionAccessToken, maximumScreenSpaceError} = selectedExample;
-    const tilesetUrl = `${TILESET_SERVER_URL}/${ionAssetId}/tileset.json`;
+    const {ionAssetId, ionAccessToken, maximumScreenSpaceError, tilesetUrl} = selectedExample;
+    const dataUrl = ionAssetId? `${TILESET_SERVER_URL}/${ionAssetId}/tileset.json` : tilesetUrl;
```

Comment:
Could this check be moved into the CesiumIonLoader, so that the application doesn't have to do this check? @belom88 @dryabinin-actionengine your thoughts?

Reply:
I'll create an issue

Reply:
I understand now that URL is used on the @deck.gl side. We cannot transform it in the `parse` function